features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1947,7 +1947,7 @@
   "organization_management.self_service.default_token_expiry_time": "7200",
   "organization_management.self_service.internal_role_name": "B2B-SS-System-Role",
   "organization_management.self_service.user_store_name_for_system_user": "PRIMARY",
-  "organization_management.organization_versioning.new_organization_version": "v0.0.0",
+  "organization_management.organization_versioning.new_organization_version": "v1.0.0",
```

Comment:
If the default version for new root organizations is updated to v1, the tests will have to be updated as in PR [1] to skip checking that the claims are not shared for new root organizations as well.

[1] https://github.com/wso2/product-is/pull/25010